apps/events/dashboard/utils.py

```diff
@@ -77,6 +78,11 @@ def handle_add_attendee(event, user_id):
     attendee.save()
 
     resp['message'] = '%s ble meldt på %s' % (user.get_full_name(), event)
+    if Payment.objects.filter(object_id=event.attendance_event.pk).count() != 0:
```

Comment:
Seems like a better solution!

Reply:
I did not mean to not add more of a comment on it, sorry about that.

I have not tested the above example, but it _should_ do exactly the same. I don't know if it fits inside the line length of our settings though 🙃

Reply:
Just thought of a way to do this a bit safer. Since this is using a generic relation there can be many Payment objects related to a single `object_id`, but for different kinds of models.

We can much rather just use:
``` python
resp['is_payment_event'] = event.attendance_event.payment is not None
```

Or something similar to check if it is a payment event

Reply:
I agree that seems safer. It is updated now.